packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -135,8 +135,8 @@ $ds-c-inset-border-width: $spacer-half;
     opacity: 1;
   }
   &::before {
-    background-color: $color-primary;
-    border-color: $color-primary-darker;
+    background-color: $choice-checked-background-color;
+    border-color: $choice-checked-background-color;
```

Comment:
Shouldnt this be `$choice-border-color`?

Reply:
previously `$choice-checked-background-color` was used https://github.com/CMSgov/design-system/blame/bb15cccb521dc283ae53577906ac83f0a83b0159/packages/design-system/src/styles/components/_Choice.scss#L104

`$choice-border-color` is `$color-base` and is used for when checkboxes are not checked